From 2d1ce7c8db81aab60d4d3b57d5df7e5fda4984d8 Mon Sep 17 00:00:00 2001
From: Raphael Mounier <mounierr07@gmail.com>
Date: Sat, 29 Apr 2023 15:41:10 +0200
Subject: [PATCH 06/13] Allow servicemanager to access hal_audio_sysbta

Allow servicemanager to access hal_audio_sysbta
Allow hal_audio_sysbta to read prop
---
 sepolicy/hal_audio_sysbta.te | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/sepolicy/hal_audio_sysbta.te b/sepolicy/hal_audio_sysbta.te
index d871ed3..d227c24 100644
--- a/sepolicy/hal_audio_sysbta.te
+++ b/sepolicy/hal_audio_sysbta.te
@@ -8,3 +8,10 @@ hal_client_domain(hal_audio_sysbta, hal_allocator)
 
 # allow audioserver to call hal_audio dump with its own fd to retrieve status
 allow hal_audio_sysbta audioserver:fifo_file write;
+
+# Allow servicemanager to access hal_audio_sysbta
+allow servicemanager hal_audio_sysbta:dir { search };
+allow servicemanager hal_audio_sysbta:file { open read };
+allow servicemanager hal_audio_sysbta:process { getattr };
+
+get_prop(hal_audio_sysbta, vendor_default_prop);
-- 
2.33.0.windows.2

